package exception

const EX_UNMATCHED_MERGE="exception.manipulate.merger.unmatch"
